.azure/rewritePackageJson.js

```diff
@@ -18,44 +18,17 @@
 const fs = require('fs');
 const process = require('process');
 
-console.log('rewriting package json');
-
-const packageJson = JSON.parse(fs.readFileSync('./package.json'));
+console.log('calling rewriting package json');
 
 if (process.argv.includes('publish')) {
-  packageJson.production = true;
-  packageJson.activationEvents = ['*'];
-} else {
-  if (packageJson.activationEvents.length > 1) {
-    throw new Error('Activation events should be * when checked in');
-  }
-
-  if (packageJson.engines.vscode !== '^1.40.0') {
-    throw new Error('Engine vscode should be ^1.40.0 when checked in');
-  }
-
-  packageJson.activationEvents = [];
+  const packageJson = JSON.parse(fs.readFileSync('./package.json'));
 
-  packageJson.actualActivationEvents.onView.forEach((event) => {
-    packageJson.activationEvents.push('onView:' + event);
-  });
-
-  packageJson.actualActivationEvents.onCommand.forEach((event) => {
-    packageJson.activationEvents.push('onCommand:' + event);
-  });
-
-  packageJson.actualActivationEvents.other.forEach((event) => {
-    packageJson.activationEvents.push(event);
-  });
-
-  packageJson.engines = {
-    vscode: '^1.40.0'
-  };
-}
+  packageJson.production = true;
 
+  const packageJsonString = JSON.stringify(packageJson, null, 4);
 
-const packageJsonString = JSON.stringify(packageJson, null, 4);
+  fs.writeFileSync('./package.json', packageJsonString, 'utf8');
 
-fs.writeFileSync('./package.json', packageJsonString, 'utf8');
+  console.log('finished rewriting package json');
 
-console.log('finished rewriting package json');
+}
```

Comment:
Should there be a new line at the end of this file?

Reply:
Don't think it matters - our linting rules don't complain or anything.